packages/react/src/components/Table/TableHead/FilterHeaderRow/FilterHeaderRow.test.jsx

```diff
@@ -1034,4 +1034,16 @@ describe('FilterHeaderRow', () => {
       expect(screen.getByTestId('filter-row-icon')).toBeVisible();
     });
   });
+
+  it('should display custom input when not undefined', () => {
+    render(
+      <FilterHeaderRow
+        {...commonFilterProps}
+        ordering={[{ columnId: 'col1' }, { columnId: 'col2' }]}
+        columns={[{ id: 'col1', customInput: 'customInput' }, { id: 'col2' }]}
```

Comment:
You are adding a string here which would set off the prop type warning since it is expecting an element.  You can either pass in rendered output (`<div>customInput</div>`) or the output of a React component (`<MyElement/>`)